docs/source/tutorials/playing-command.rst

```diff
@@ -76,22 +84,35 @@ Now the command will be able to react a bit more to your user::
     <YourNick> .hello noon
     <Sopel> Hello YourNick, are you having lunch?
 
+How does that work? Well, the short version is that Sopel uses regex
+(`Regular Expression`__) to match a message to a plugin callable, and the
+``trigger`` object exposes the match result.
+
 .. seealso::
 
     You can learn much more about the :class:`~sopel.plugin.command` decorator
     by reading its documentation.
 
+.. note::
+
+    In the case of a command, the regex is entirely managed by Sopel itself,
+    while the generic :func:`@plugin.rule <sopel.plugin.rule>` decorator
+    allow you to define your own regex.
+
+.. __: https://en.wikipedia.org/wiki/Regular_expression
+
+
 And... action!
 ==============
 
-Some users say ``.hello`` out loud, and other will say it with an action. How
+Some users say ``.hello`` out loud, and others will say it with an action. How
 do you react to these? Let's go back to our initial example of the ``shrug``
 command: it uses a second decorator, ``action_command('shrugs')``, with a
 different name. How does that work?
 
 Sopel knows how to register the same plugin callable for different types of
 trigger, so both ``.shrug`` and ``/me shrugs`` work. For example, you could do
-that for your hello plugin::
+this for your hello plugin::
 
     @plugin.command('hello')
     @plugin.action_command('hi')
```

Comment:
```suggestion
    @plugin.action_command('says hi')
```

Don't see any reason this wouldn't work, and it's a better demonstration of how people generally define action commands (i.e. as _actions_ that sound like natural language).

Reply:
Hm... what about this instead?

```suggestion
    @plugin.action_command('waves')
```